src/components/auth/Auth.js

```diff
@@ -0,0 +1,64 @@
+import auth0 from 'auth0-js';
+
+class Auth {
+  constructor() {
+    this.auth0 = new auth0.WebAuth({
+      // the following three lines MUST be updated
+      domain: 'coding-coach.eu.auth0.com',
+      audience: 'https://coding-coach.eu.auth0.com/userinfo',
+      clientID: 'bzQKiL4VQtjla1TeIgizh9YIoTyn01Zk',
```

Comment:
Please do not commit secret keys... Pick it from the env file which should be in git ignore

Reply:
This isn’t for committing. It’s just an example